requirements.txt

```diff
@@ -7,3 +7,7 @@ jinja2
 PyYAML
 cryptography
 packaging
+# NOTE: resolvelib 0.x version bumps should be considered major/breaking
+# NOTE: and we should update the upper cap with care, at least until 1.0
+# NOTE: Ref: https://github.com/sarugaku/resolvelib/issues/69
+resolvelib >= 0.5.3, < 0.6.0  # New dependency resolver that pip>=20.3 uses
```

Comment:
Constraints included here need to be in `test/lib/ansible_test/_data/requirements/constraints.txt` also.

Reply:
Oh, I missed that. Should we have some sanity check for this?

Reply:
No, because we haven't had any constraints in our requirements file.

We'll probably want to add one that makes sure any constraints here match the constraints file.